code/src/index.css

```diff
@@ -1,13 +1,208 @@
 body {
-  margin: 0;
+ 
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
     "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-image: url("/assets/background.jpg");
+  background-size: cover;
+  
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
+  font-family: 'Roboto Mono', monospace;
+
+}
+
+.body{
+  width: 100%;
+  
+}
+
+/*Loading spinner*/
+
+.loading-overlay {
+position: absolute;
+top: 0;
+bottom: 0;
+left: 0;
+right: 0;
+background-color: honeydew;
+display: flex;
+justify-content: center;
+align-items: center;
+}
+
+.loading-overlay img {
+  height: 300px;
+  width: 300px;
+  animation: spinner infinite 0.8s;
+}
+
+@keyframes spinner {
+  0% {
+    transform: rotate(0);
+  }
+  50% {
+    transform: rotate(180deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
+h1 {
+  font-family: 'Roboto Mono', monospace;
+  font-size: 60px;
+  font-weight: 700;
+  text-shadow: 3px 3px 3px black;
+  text-align: center;
+  margin: 30px;
+  color:white;
+}
+/*Containers*/
+
+.form-container{
+  width: 100%;
+  margin: 40px auto;
+  border: 1px solid black;
+  box-shadow: 13px 13px black;
+  display: flex;
+  flex-direction: column;
+  background-color: #F3F0F0;
+
+}
+.message-container{
+  width: 100%;
+  margin: 40px auto;
+  border: 1px solid black;
+  box-shadow: 13px 13px black;
+  background-color: white;
+}
+/*Form for thought-submission*/
+
+.label-form {
+  margin: 25px 0 15px 25px;
+  font-weight: 600;
+}
+/*This is where you type in your thought*/
+.thought-input {
+  border: 1px solid #D5D2D2;
+  margin: 0 auto;
+  width: 90%;
+  height: 70px;
+  font-family: 'Roboto Mono', monospace;
+  padding: 10px;
+}
+/*This is the button-character counter wrapper*/
+.button-count{
```

Comment:
on the smallest mobile size (320px) the character-count pops outside the card and on 375px there is no padding between it and the button. Maybe you want to add a set width here or maybe just add a media query for the smaller devices :)